packages/core-transaction-pool/src/memory.ts

```diff
@@ -242,4 +273,12 @@ export class Memory {
 
         return removed;
     }
+
+    private currentHeight(): number {
```

Comment:
After the introduction of core-state in https://github.com/ArkEcosystem/core/commit/a4567cd2696f2c9c6061c47dc0172dc31d994417 the last block is always present before the transaction pool mounts because `core-database-postgres` mounts before `core-transaction-pool` which will set the last block.

Instead of calling this method you can now do `app.resolvePlugin("state").getLastHeight()` to get the last height.

Reply:
Y